fortran/src/cudafor.f90

```diff
@@ -15,7 +15,7 @@ module cudaFor
 public :: cuda_stream_kind
 public :: cudaGetDeviceCount
 public :: cudaSetDevice
-public :: cudaMAlloc
+public :: cudaMalloc
 public :: cudaMemCpy
```

Comment:
You would probably want to change `cudaMemCpy` to `cudaMemcpy` and `ncclBCast` to `ncclBcast` throughout.